src/konfetti/vault/asynchronous.py

```diff
@@ -32,6 +33,14 @@ def cached_call(method):
 class AsyncVaultBackend(BaseVaultBackend):
     is_async = True
 
+    import aiohttp
```

Comment:
`aiohttp` is an optional dependency, it could be not available in all cases e.g. on Python 3.7 with `pip install konfetti`